.circleci/config.yml

```diff
@@ -141,6 +141,17 @@ jobs:
             cp junit.xml test-results/npm
       - store_test_results:
           path: test-results
+  eslint:
+    executor: encoded-executor
+    resource_class: medium
+    steps:
+      -  buildout
```

Comment:
I tried an image. I got issues with python needing to be installed. So, to keep things simple, I used "buildout". There does no seems to considerable lag